packages/manager/modules/pci/src/projects/project/private-registry/create/create.controller.js

```diff
@@ -7,10 +7,12 @@ export default class {
     this.privateRegistryService = pciPrivateRegistryService;
     this.loading = false;
     this.registry = {};
+    this.selectedPlan = null;
   }
 
   create() {
     this.loading = true;
+    this.trackClick(`PRIVATEREGISTRY_CREATE_PLAN_CREATE_${this.selectedPlan}`);
```

Comment:
I see this code in multiple places. Can you move this line to a separate method (call it trackPrivateRegistryCreate() etc) and call it whenever required.

Reply:
Looks cleaner now